src/Altinn.Profile/Controllers/TriggerController.cs

```diff
@@ -27,13 +32,24 @@ public class TriggerController(IContactRegisterUpdateJob contactRegisterUpdateJo
     /// <returns>
     /// A task that represents the asynchronous operation. If successful, returns a status 200 OK.
     /// </returns>
-    /// <response code="200">Starting the synchronisation work was successfull.</response>
+    /// <response code="200">Starting the synchronization work was successful.</response>
+    /// <response code="500">An error occurred while starting the synchronization.</response>
     [HttpGet("syncpersonchanges")]
     [ProducesResponseType(StatusCodes.Status200OK)]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError)]
     public ActionResult SyncChanges()
     {
-        _contactRegisterUpdateJob.SyncContactInformationAsync();
+        try
+        {
+            _contactRegisterUpdateJob.SyncContactInformationAsync();
```

Comment:
You must await

Reply:
I didn't use `await` in the `SyncChanges` method because the goal is to immediately return a response indicating that synchronization has started, without waiting for the process to complete. This is achieved using a *fire-and-forget* approach (`Task.Run`). The task will run in the background.

Reply:
This function is simplified so that only one instance of the synchronization task will be executed at a time.